wazuh/wazuh-kibana-app
# Fixed propagation event for flyout on decoders

### Description
Fixed flyout showing information related to a decoder showing information about another decoder when clicking on the button/link to open the content of the file containing the decoder.
 
### Issues Resolved
- #5219 

### Evidence

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/b0013faf-0c29-4217-bcbb-52074cd70003)


### Test
1. Navigate to Management/Decoders
2. Click on any row of the decoders table. A flyout will be opened with data from the selected decoder.
3. In the table of Related decoders clicks on a button/link of File column.
4. Check that the flyout data is not changed before closing

### Check List
